amundsen_application/static/js/components/common/Breadcrumb/index.tsx

```diff
@@ -1,14 +1,30 @@
 import * as React from 'react';
+import { connect } from 'react-redux';
 import { Link } from 'react-router-dom';
 
 import './styles.scss';
+import { GlobalState } from 'ducks/rootReducer';
 
-export interface BreadcrumbProps {
-  path: string;
-  text: string;
+export interface OwnProps {
+  path?: string;
+  text?: string;
 }
 
-const Breadcrumb: React.SFC<BreadcrumbProps> = ({ path, text }) => {
+export interface StateFromProps {
+  searchTerm: string;
+}
+
+export type BreadcrumbProps = OwnProps & StateFromProps;
+
+export const Breadcrumb: React.SFC<BreadcrumbProps> = (props) => {
```

Comment:
nit: If you do:
`export const Breadcrumb: React.SFC<BreadcrumbProps> = ({ path, text, searchTerm ) => {`
then variables `path` and `text` and `searchTerm` will automatically be defined and pulled from the `props`. Then, you can also set the default `path` and `text` in the `defaultProps` on line 40. This will make it such that all you have to have in this section is:
```
if (searchTerm) {
    path = `/search?searchTerm=${props.searchTerm}&selectedTab=table&pageIndex=0`
    text = 'Search Results'
}
```

Reply:
:+1: yeah that's way cleaner